classes/class4/homework/routes/cats.js

```diff
@@ -0,0 +1,95 @@
+module.exports = function(){
+    var express = require('express');
+    var router = express();
+    var Cat = require('../models/cat_model.js')
+
+    router.get('/', function(req,res,next) {
+      Cat.find({}).sort({age: -1}).exec(function(err,cats){
+        if (err) {
+            console.log('Error displaying all cats!');
+            res.status(404).send('Error displaying all cats!');
+        } else {
+            cats = cats.map(function (cat){
+                return 'name: '+cat['name']+' age: '+cat['age']+' colors: '+cat['colors'];
+            });
+            res.render('home',{'cats': cats});
```

Comment:
You aren't actually using handlebars properly. You want to just send the cat array over and render the appropriate information in handlebars. Something like this: 
{{#each cats}}
  <li>{{name}}:
    <ul>
      <li>age: {{age}}</li>
      <li>color(s): {{#each colors}}{{this}}{{/each}}</li>
    </ul>
  </li>
  {{/each}}